demos/interactive_face_detection_demo/README.md

```diff
@@ -3,20 +3,21 @@
 This demo showcases Object Detection task applied for face recognition using sequence of neural networks.
 Async API can improve overall frame-rate of the application, because rather than wait for inference to complete,
 the application can continue operating on the host while accelerator is busy.
-This demo executes four parallel infer requests for the Age/Gender Recognition, Head Pose Estimation, Emotions Recognition, and Facial Landmarks Detection networks that run simultaneously. You can use a set of the following pre-trained models with the demo:
+This demo executes five parallel infer requests for the Age/Gender Recognition, Head Pose Estimation, Emotions Recognition, Facial Landmarks Detection and Antispoofing Classificator networks that run simultaneously. You can use a set of the following pre-trained models with the demo:
```

Comment:
```suggestion
This demo executes five parallel infer requests for the Age/Gender Recognition, Head Pose Estimation, Emotions Recognition, Facial Landmarks Detection and Antispoofing Classifier networks that run simultaneously. You can use a set of the following pre-trained models with the demo:
```

Reply:
@kirillProkofiev please provide comments if something was changed, We should not look through your latest commits in order to understand if you addressed raised questions.

Reply:
next time I will know, thanks.
@IRDonch  All issues were resolved